domain/src/test/kotlin/com/antosik/benchproject/domain/favourite/usecase/GetFavouriteMoviesUseCaseTest.kt

```diff
@@ -0,0 +1,33 @@
+package com.antosik.benchproject.domain.favourite.usecase
+
+import com.antosik.benchproject.domain.favourite.repo.FavouriteMoviesRepository
+import com.antosik.benchproject.domain.popular.movies.entity.MovieModel
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.test.runTest
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.jupiter.api.DynamicTest
+import org.junit.jupiter.api.TestFactory
+
+internal class GetFavouriteMoviesUseCaseTest {
+
+    val favouriteMoviesRepository: FavouriteMoviesRepository = mockk()
+    val tested = GetFavouriteMoviesUseCase(favouriteMoviesRepository)
+
+    @TestFactory
+    fun `SHOULD return expected favorite movies WHEN specific data incoming from repository`() =
```

Comment:
do we really need a dynamic test here ? do we have any logic that is different when the list is empty or not ?

Reply:
To be honest we didn't have any logic when the list is empty or not. I would like just to test both cases but if you think one test will be enough I can go with it.

Reply:
same as in previous comment :)

Reply:
Done 😄